sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_policies.py

```diff
@@ -28,18 +28,18 @@ def on_response(self, request, response):
             data = ContentDecodePolicy.deserialize_from_http_generics(
                 response.http_response
             )
-            if self._is_lro and (not data or data.get("status") not in _FINISHED):
+            if self._is_lro and (not data or data.get("status", "").lower() not in _FINISHED):
```

Comment:
noticed we should be checking lower() here in the case of something like "partiallySucceeded"